scripts/build.js

```diff
@@ -4,6 +4,8 @@
 process.env.BABEL_ENV = 'production';
 process.env.NODE_ENV = 'production';
 
+process.env.REACT_APP_VERSION = process.env.REACT_APP_VERSION || 'a.b.c';
```

Comment:
use `process.env.npm_package_version` which is a standard node env variable loaded from `package.json`